src/openfermion/ops/_interaction_rdm.py

```diff
@@ -17,10 +17,10 @@
 import numpy
 
 from openfermion.ops import (FermionOperator,
-                             PolynomialTensor,
                              InteractionOperator,
-                             normal_ordered,
-                             QubitOperator)
+                             PolynomialTensor,
+                             QubitOperator,
+                             normal_ordered)
```

Comment:
Is this out of alphabetical order to avoid a circular import issue?

Reply:
Capital letters come before lower case letters (at least according to Pythons `ord` function :P)